### PR TITLE
Fix all found typos

### PR DESCRIPTION
**Pull request by williamdes**
With the help of https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

I reviewed all files except the folder `_posts`.

### CODE REVIEW

File: doc/faq.md [hunk 1]
```diff
@@ -328,11 +328,11 @@ If your issue is related to crashes, then you need to obtain core file prior to
 
 
 
-For issues about the rules, we usually need a **message sample** that causes a problem. To protect your privacy, you can remove unrelevant headers and content. E.g. you can blind message sender/recipients, subject and/or other fields.
+For issues about the rules, we usually need a **message sample** that causes a problem. To protect your privacy, you can remove irelevant headers and content. E.g. you can blind message sender/recipients, subject and/or other fields.
```
Comment (moisseev) [inline doc/faq.md:331]: I believe the English spelling is "irrelevant".

File: contribution.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ The original Lua bindings are too complicated at the moment. The idea is to unif
 
 ### Additional statistics backends
 
-Currently, we support plain mmapped files only for storing statistics tokens. However, we'd like to support other backends, such as redis, sqlite or other storages.
```
Comment (fatalbanana) [inline contribution.md:33]: Probably it should be spelled `memory-mapped`.

It is of course ancient history.

Comment (williamdes) [reply contribution.md:33]: Thanks, this one the one change I was not sure about

File: doc/faq.md (hunk 1 above)
Comment (williamdes) [reply doc/faq.md:331]: thanks, indeed, I will change this

File: contribution.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ The original Lua bindings are too complicated at the moment. The idea is to unif
...
+Currently, we support plain mapped files only for storing statistics tokens. However, we'd like to support other backends, such as redis, sqlite or other storages.
```
Comment (williamdes) [inline contribution.md:33]:
```suggestion
Currently, we support plain memory-mapped files only for storing statistics tokens. However, we'd like to support other backends, such as redis, sqlite or other storages.
```

File: doc/faq.md (hunk 1 above)
Comment (williamdes) [inline doc/faq.md:331]:
```suggestion
For issues about the rules, we usually need a **message sample** that causes a problem. To protect your privacy, you can remove irrelevant headers and content. E.g. you can blind message sender/recipients, subject and/or other fields.
```

**Review by moisseev — APPROVED** [review]

**Pull request merged**